networking/advertise-service-ips.md

```diff
@@ -142,6 +144,82 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
    For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
 
+#### Advertise service load balancer IP addresses
+
+The following steps will configure {{site.prodname}} to advertise Service `status.LoadBalancer.Ingress.IP` addresses.
+
+1. Identify the IP ranges to be used for Service LoadBalancer address allocation.
+
+1. Check to see if you have a default BGPConfiguration.
+
+   ```bash
+   calicoctl get bgpconfig default
+   ```
+
+1. Based on above results, update or create a BGPConfiguration.
+
+   **Update default BGPConfiguration**
+   Patch the BGPConfiguration using the following command, adding your own service load balancer IP CIDRs:
+
+   ```bash
+   calicoctl patch BGPConfig default --patch '{"spec": {"serviceLoadBalancer": [{"cidr": "x.x.x.x/16"}]}}'
+   ```
+
+   **Create default BGPConfiguration**
+   Use the following sample command to create a default BGPConfiguration. Add your CIDR blocks for load balancer IPs to be advertised in the `serviceLoadBalancerIPs` field.
+
+   ```bash
+   calicoctl create -f - <<EOF
+   apiVersion: projectcalico.org/v3
+   kind: BGPConfiguration
+   metadata:
+     name: default
+   spec:
+     serviceLoadBalancerIPs:
+     - cidr: x.x.x.x/16
+   EOF
+   ```
+
+   For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
+
+Allocation of Service LoadBalancer addresses is outside the current scope of {{site.prodname}}, but can be implemented with an external controller.
+You may build your own, or use a third-party implementation such as that provided by the MetalLB project.
+
+To install the MetalLB controller for allocating addresses, perform the following steps.
+
+1. Follow [the MetalLB documentation](https://metallb.universe.tf/installation/#installation-by-manifest) to install the `metallb-system/controller` resources. However, do not install the `metallb-system/speaker` component.
+
+1. Configure MetalLB to provision addresses from the CIDR range given to {{site.prodname}} above by creating the following config map, replacing `x.x.x.x/16` with your chosen CIDR.
```

Comment:
Same here...had to read sentence twice to get it. how about:

Configure MetalLB to provision addresses (CIDR range in your Calico BGPConfiguration) by creating the following config map, replacing x.x.x.x/16 with your chosen CIDR.